src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/test_rdbms_commands.py

```diff
@@ -77,19 +81,21 @@ def _test_server_mgmt(self, database_engine, resource_group_1, resource_group_2)
                    self.create_random_name('azuredbclirestore', SERVER_NAME_MAX_LENGTH)]
         admin_login = 'cloudsa'
         admin_passwords = ['SecretPassword123', 'SecretPassword456']
-        edition = 'Basic'
-        old_cu = 100
-        new_cu = 50
+        edition = 'GeneralPurpose'
+        old_cu = 2
+        new_cu = 4
+        family = 'Gen4'
+        skuname = '{}_{}_{}'.format("GP", family, old_cu)
 
         rg = resource_group_1
-        loc = 'westeurope'
+        loc = 'brazilsouth'
 
         # test create server
         self.cmd('{} server create -g {} --name {} -l {} '
```

Comment:
Highly recommended to use the kwarg style syntax for tests, for readability.  https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md#sample-5-get-more-from-resourcegrouppreparer

Reply:
The doc has been updated based on your feedback.